CI failures with pre-reef and main jobs

```
#6 27.40 Error: 
#6 27.40  Problem: problem with installed package curl-minimal-7.76.1-29.el9.x86_64
#6 27.40   - package curl-minimal-7.76.1-29.el9.x86_64 from @System conflicts with curl provided by curl-7.76.1-29.el9.x86_64 from baseos
#6 27.40   - package curl-minimal-7.76.1-22.el9.x86_64 from baseos conflicts with curl provided by curl-7.76.1-29.el9.x86_64 from baseos
#6 27.40   - package curl-minimal-7.76.1-23.el9.x86_64 from baseos conflicts with curl provided by curl-7.76.1-29.el9.x86_64 from baseos
#6 27.40   - package curl-minimal-7.76.1-26.el9.x86_64 from baseos conflicts with curl provided by curl-7.76.1-29.el9.x86_64 from baseos
#6 27.40   - package curl-minimal-7.76.1-28.el9.x86_64 from baseos conflicts with curl provided by curl-7.76.1-29.el9.x86_64 from baseos
#6 27.40   - package curl-minimal-7.76.1-29.el9.x86_64 from baseos conflicts with curl provided by curl-7.76.1-29.el9.x86_64 from baseos
#6 27.40   - package curl-minimal-7.76.1-12.el9.x86_64 from appstream conflicts with curl provided by curl-7.76.1-29.el9.x86_64 from baseos
#6 27.40   - package curl-minimal-7.76.1-14.el9.x86_64 from appstream conflicts with curl provided by curl-7.76.1-29.el9.x86_64 from baseos
#6 27.40   - cannot install the best candidate for the job
#6 27.40 (try to add '--allowerasing' to command line to replace conflicting packages or '--skip-broken' to skip uninstallable packages or '--nobest' to use not only best candidate packages)
#6 ERROR: process "/bin/sh -c true &&     echo \"Check: [ ${CEPH_VERSION} = ${GO_CEPH_VERSION} ]\" &&     [ \"${CEPH_VERSION}\" = \"${GO_CEPH_VERSION}\" ] &&     yum update -y --disablerepo=ganesha &&     cv=\"$(rpm -q --queryformat '%{version}-%{release}' ceph-common)\" &&     yum install -y     git wget curl make     /usr/bin/cc /usr/bin/c++     \"libcephfs-devel-${cv}\" \"librados-devel-${cv}\" \"librbd-devel-${cv}\"     gdb libcephfs2-debuginfo librados2-debuginfo librbd1-debuginfo &&     yum clean all &&     true" did not complete successfully: exit code: 1
------
 > [2/6] RUN true &&     echo "Check: [ main = main ]" &&     [ "main" = "main" ] &&     yum update -y --disablerepo=ganesha &&     cv="$(rpm -q --queryformat '%{version}-%{release}' ceph-common)" &&     yum install -y     git wget curl make     /usr/bin/cc /usr/bin/c++     "libcephfs-devel-${cv}" "librados-devel-${cv}" "librbd-devel-${cv}"     gdb libcephfs2-debuginfo librados2-debuginfo librbd1-debuginfo &&     yum clean all &&     true:
27.40   - package curl-minimal-7.76.1-29.el9.x86_64 from @System conflicts with curl provided by curl-7.76.1-29.el9.x86_64 from baseos
27.40   - package curl-minimal-7.76.1-22.el9.x86_64 from baseos conflicts with curl provided by curl-7.76.1-29.el9.x86_64 from baseos
27.40   - package curl-minimal-7.76.1-23.el9.x86_64 from baseos conflicts with curl provided by curl-7.76.1-29.el9.x86_64 from baseos
27.40   - package curl-minimal-7.76.1-26.el9.x86_64 from baseos conflicts with curl provided by curl-7.76.1-29.el9.x86_64 from baseos
27.40   - package curl-minimal-7.76.1-28.el9.x86_64 from baseos conflicts with curl provided by curl-7.76.1-29.el9.x86_64 from baseos
27.40   - package curl-minimal-7.76.1-29.el9.x86_64 from baseos conflicts with curl provided by curl-7.76.1-29.el9.x86_64 from baseos
27.40   - package curl-minimal-7.76.1-12.el9.x86_64 from appstream conflicts with curl provided by curl-7.76.1-29.el9.x86_64 from baseos
27.40   - package curl-minimal-7.76.1-14.el9.x86_64 from appstream conflicts with curl provided by curl-7.76.1-29.el9.x86_64 from baseos
27.40   - cannot install the best candidate for the job
27.40 (try to add '--allowerasing' to command line to replace conflicting packages or '--skip-broken' to skip uninstallable packages or '--nobest' to use not only best candidate packages)
------
Dockerfile:11
--------------------
  10 |     
  11 | >>> RUN true && \
  12 | >>>     echo "Check: [ ${CEPH_VERSION} = ${GO_CEPH_VERSION} ]" && \
  13 | >>>     [ "${CEPH_VERSION}" = "${GO_CEPH_VERSION}" ] && \
  14 | >>>     yum update -y --disablerepo=ganesha && \
  15 | >>>     cv="$(rpm -q --queryformat '%{version}-%{release}' ceph-common)" && \
  16 | >>>     yum install -y \
  17 | >>>     git wget curl make \
  18 | >>>     /usr/bin/cc /usr/bin/c++ \
  19 | >>>     "libcephfs-devel-${cv}" "librados-devel-${cv}" "librbd-devel-${cv}" \
  20 | >>>     gdb libcephfs2-debuginfo librados2-debuginfo librbd1-debuginfo && \
  21 | >>>     yum clean all && \
  22 | >>>     true
  23 |     
--------------------
ERROR: failed to solve: process "/bin/sh -c true &&     echo \"Check: [ ${CEPH_VERSION} = ${GO_CEPH_VERSION} ]\" &&     [ \"${CEPH_VERSION}\" = \"${GO_CEPH_VERSION}\" ] &&     yum update -y --disablerepo=ganesha &&     cv=\"$(rpm -q --queryformat '%{version}-%{release}' ceph-common)\" &&     yum install -y     git wget curl make     /usr/bin/cc /usr/bin/c++     \"libcephfs-devel-${cv}\" \"librados-devel-${cv}\" \"librbd-devel-${cv}\"     gdb libcephfs2-debuginfo librados2-debuginfo librbd1-debuginfo &&     yum clean all &&     true" did not complete successfully: exit code: 1
make: *** [Makefile:245: .build.main] Error 1
Error: Process completed with exit code 2.
```

https://github.com/ceph/go-ceph/actions/runs/9326472027/job/25675028122 (main)
https://github.com/ceph/go-ceph/actions/runs/9326472027/job/25675027834 (pre-reef)

--- a/testing/containers/ceph/Dockerfile
+++ b/testing/containers/ceph/Dockerfile
@@ -14,7 +14,7 @@ RUN true && \
     yum update -y --disablerepo=ganesha && \
     cv="$(rpm -q --queryformat '%{version}-%{release}' ceph-common)" && \
     yum install -y \
-    git wget curl make \
+    git wget /usr/bin/curl make \
     /usr/bin/cc /usr/bin/c++ \
     "libcephfs-devel-${cv}" "librados-devel-${cv}" "librbd-devel-${cv}" \
     gdb libcephfs2-debuginfo librados2-debuginfo librbd1-debuginfo && \

